repos/autodrive.repos

```diff
@@ -0,0 +1,6 @@
+repositories:
+    src/external/ackermann_odom:
```

Comment:
change the directory to `src/external/actuator/ackermann-odom-ros2`
Please add annotation in https://github.com/chenyenru/ackermann-odom-ros2.git 
If you used code from other organization

Reply:
```
Starting >>> ackermann_odom
--- stderr: ackermann_odom                           
CMake Error at CMakeLists.txt:17 (find_package):
  By not providing "Findbackward_ros.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "backward_ros", but CMake did not find one.

  Could not find a package configuration file provided by "backward_ros" with
  any of the following names:

    backward_rosConfig.cmake
    backward_ros-config.cmake

  Add the installation prefix of "backward_ros" to CMAKE_PREFIX_PATH or set
  "backward_ros_DIR" to a directory containing one of the above files.  If
  "backward_ros" provides a separate development package or SDK, be sure it
  has been installed.


---
Failed   <<< ackermann_odom [17.6s, exited with code 1]
```

Reply:
please remove this dependency if you dont need it

Reply:
```
[ERROR] [ackermann_odom_node-4]: exception occurred while executing process:
Traceback (most recent call last):
  File "/opt/ros/foxy/lib/python3.8/site-packages/launch/actions/execute_process.py", line 723, in __execute_process
    transport, self._subprocess_protocol = await async_execute_process(
  File "/opt/ros/foxy/lib/python3.8/site-packages/osrf_pycommon/process_utils/async_execute_process_asyncio/impl.py", line 142, in async_execute_process
    transport, protocol = yield from _async_execute_process_nopty(
  File "/opt/ros/foxy/lib/python3.8/site-packages/osrf_pycommon/process_utils/async_execute_process_asyncio/impl.py", line 46, in _async_execute_process_nopty
    transport, protocol = yield from loop.subprocess_exec(
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1630, in subprocess_exec
    transport = await self._make_subprocess_transport(
  File "/usr/lib/python3.8/asyncio/unix_events.py", line 197, in _make_subprocess_transport
    transp = _UnixSubprocessTransport(self, protocol, args, shell,
  File "/usr/lib/python3.8/asyncio/base_subprocess.py", line 36, in __init__
    self._start(args=args, shell=shell, stdin=stdin, stdout=stdout,
  File "/usr/lib/python3.8/asyncio/unix_events.py", line 789, in _start
    self._proc = subprocess.Popen(
  File "/usr/lib/python3.8/subprocess.py", line 858, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1704, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'gdb'
```